calour/ms1_experiment.py

```diff
@@ -91,9 +92,198 @@ def heatmap(self, *args, **kwargs):
         '''
         if 'norm' not in kwargs:
             kwargs['norm'] = mpl.colors.LogNorm()
+        if 'mz_rt' in self.feature_metadata.columns:
+            if 'yticklabel_len' not in kwargs:
+                kwargs['yticklabel_len'] = None
+            if 'feature_field' not in kwargs:
+                kwargs['feature_field'] = 'mz_rt'
+            if 'yticklabel_kwargs' not in kwargs:
+                kwargs['yticklabel_kwargs'] = {'size': 6, 'rotation': 0}
         super().heatmap(*args, **kwargs)
 
     def __repr__(self):
         '''Return a string representation of this object.'''
         return 'MS1Experiment %s with %d samples, %d features' % (
             self.description, self.data.shape[0], self.data.shape[1])
+
+    def get_spurious_duplicates(self, mz_tolerance=0.001, rt_tolerance=2, corr_thresh=0.8, inplace=False, negate=False):
```

Comment:
this function should use your new `filter_mz_rt` function? you have redundant code in the 2 functions.